learning_journal/models.py

```diff
@@ -25,3 +28,28 @@ class MyModel(Base):
     value = Column(Integer)
 
 Index('my_index', MyModel.name, unique=True, mysql_length=255)
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = Column(Integer, primary_key=True)
+    title = Column(Unicode(255), unique=True, nullable=False)
+    body = Column(Text)
+    created = Column(DateTime, default=datetime.datetime.utcnow)
+    edited = Column(DateTime, default=datetime.datetime.utcnow)
+    
+    @classmethod
+    def all(class_):
+        """Return a query of all entries."""
+        Entry = class_
+        q = DBSession.query(Entry)
+        q = q.order_by(Entry.created)
```

Comment:
Does this really give you the order you want?  Remember that you should have the entries ordered with the most recent first.